client/state/current-user/selectors.js

```diff
@@ -104,7 +104,7 @@ export const getCurrentUserEmail = createCurrentUserSelector( 'email' );
  * @return {?Boolean}            Whether capability name is valid
  */
 export function isValidCapability( state, siteId, capability ) {
-	const capabilities = state.currentUser.capabilities[ siteId ];
+	const capabilities = get( state, [ 'currentUser', 'capabilities', siteId ] );
```

Comment:
The reasons this was used was to not require to going to all automated tests and adding some user state.  I give more details about this in the description.